### ISSUE TITLE
Sodium crash on new update

### ISSUE THREAD

**Issue by LdragoLegendOP** [PojavLauncherTeam/PojavLauncher#4657]
### Describe the bug

I'm join some server with sodium and 20-30 minit my game crash on latest update

### The log file and images/videos

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/12773300/latestlog.txt)

### Steps To Reproduce

```markdown
Plzz reply me
```

### Expected Behavior

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/12773307/latestlog.txt)

### Platform

```markdown
- Device model:poco c3
- CPU architecture:32
- Android version:10 
- PojavLauncher version: new version
```

### Anything else?

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/12773305/latestlog.txt)

**Comment by FarnGitHub** [comment]
Sodium is no longer supported by pojavlauncher in this new update . Sadly

**Comment by GaucovyAero** [comment]
sodium implemented features that killed off pojav after 0.5.0+, they wont support pojav
